### DISCUSSION TITLE
How to use  "privateFieldConfig"  attribute in the schema

### DISCUSSION THREAD

**Discussion by AbhiGaddi** [Sunbird-RC/community#788] [General]
We are currently exploring of masking PII data for specific fields. We have come across the “privateFieldConfig” feature in documentation, 
Is this feature has already been released ? If it is released From which version it is available ? However, we are not clear about its usage within the schema. Could you kindly provide me with a sample schema that demonstrates the use of the “privateFieldConfig” attribute ?

**Comment by holashchand** [comment]
Hey @AbhiGaddi, masking private fields or internal fields properties used for events emitted, It doesn't do anything on how to store them, they will still be stored without any masking/hashing in the registry db

**Comment by holashchand** [comment]
Hi @AbhiGaddi, We have picked up the task to encrypt PII data.

Here is the issue created #798

Their are some open questions in there, It would be great if you could comment on those
